lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -9,6 +9,7 @@ use Encode;
 use DBI qw(:utils);
 use JSON::PP;
 use Digest::MD5 qw(md5_hex);
+use Data::Dumper;
```

Comment:
Could you remove this unused import?

There are more of these below.

Reply:
I added this one @mattias-p because surprisingly it is used in the generation of error logs (search to file for Dumper)

Reply:
@mtoma, you have added the library to `MySQL.pm`. What has changed to make it necessary to add it? MySQL backend is not the target of this PR. And if it should be there, shouldn't it be added for PostgreSQL too?

Reply:
@mtoma, right. I can see it's needed in SQLite.pm as well. I didn't expect that, though it does make sense.